infra/main.bicep

```diff
@@ -11,7 +11,7 @@ param resourceToken string = toLower(uniqueString(subscription().id, environment
 param location string
 
 @description('Name of App Service plan')
-param hostingPlanName string = '${environmentName}-hosting-plan-${resourceToken}'
+param hostingPlanName string = 'hosting-plan-${resourceToken}'
```

Comment:
How does this renaming affect resources that are already deployed?

Reply:
I believe that bicep does not manage state of changed resources, so I think that we are going to get newly create resources in peoples resource groups. I deially we need a changelog to say that breaking chnages are coming - what do you think?

Reply:
@cecheta I've pushed and only changed the app services

Reply:
It looks like a number of them are still changing?

![image](https://github.com/Azure-Samples/chat-with-your-data-solution-accelerator/assets/60179183/22ba8098-0d61-4413-8986-732e7e885c72)